CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
-# CHANGELOG
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## 1.2.0 - 2024-06-25
-[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
-## 1.1.3 -2024-02-16
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.2.0] - 2024-06-25
+### Added
+- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - Add vidarr labels to outputs (changes to medata only)
+
+## [1.1.3] -2024-02-16
+### Added
 - Added alternate solution
 
-## 1.1.2 -2023-11-03
+## [1.1.2] -2023-11-03
```

Comment:
tag 1.1.1 is missing, need to add it here. List of changes should be in PR log/Jira